src/collection-resource.js

```diff
@@ -78,9 +78,9 @@ class CollectionResource extends Resource {
    * @param {String} id The id of the collection to fetch.
    * @return {Promise|GraphModel} A promise resolving with a `GraphModel` of the collection.
    */
-  fetchWithProducts(id) {
+  fetchWithProducts(id, productsFirst = 20) {
```

Comment:
In the case the signature expands should this not be a key word arg instead of a positional param?